src/index.js

```diff
@@ -460,6 +461,33 @@ export function removeContext(sequelize) {
   shimmer.unwrap(sequelize.constructor.Association.BelongsToMany.prototype, 'get');
 }
 
+export function clearCache(context, sequelize) {
```

Comment:
considering whether or not this should be named `reset` instead since it does more than just clear out the dataloader caches.

Reply:
I can change that. Didn't have time over the weekend bc of a project. But I'll pick it up soon.